remote/src/main/scala/org/apache/pekko/remote/PhiAccrualFailureDetector.scala

```diff
@@ -153,15 +151,7 @@ class PhiAccrualFailureDetector(
         val interval = timestamp - latestTimestamp
         // don't use the first heartbeat after failure for the history, since a long pause will skew the stats
         if (isAvailable(timestamp)) {
```

Comment:
nit: Still a redundant single bracket here, i.e. should be

```scala
if (isAvailable(timestamp))
  recordInterval(interval)
else
  oldState.history
```

You can even put it on a single line

Reply:
I removed the redundant bracket, since it isn't an assignment I'd leave it on different lines for clarity